### ISSUE TITLE
Secondary icon buttons should be focusable and clickable via keyboard

### ISSUE THREAD

**Issue by AlbyIanna** [arduino/arduino-ide#1695]
### Describe the problem

Secondary buttons with icon are not focusable and therefore not usable with a keyboard 😢 
<img width="110" alt="Screenshot 2022-11-18 at 11 10 15" src="https://user-images.githubusercontent.com/6939054/202677407-b3f6318c-1778-424f-9272-07cc54384075.png">

### To reproduce

Try to focus on the contextual menu button of the tab bar.

Or the icons in the sketchbook when focusing on a sketch:
<img width="355" alt="image" src="https://user-images.githubusercontent.com/6939054/202677915-514eacdb-28a3-45eb-862f-efeab6142f44.png">

### Expected behavior

You should be able to move the focus on them and click them.

### Arduino IDE version

any

### Operating system

N/A

### Operating system version

any

### Issue checklist

- [X] I searched for previous reports in [the issue tracker](https://github.com/arduino/arduino-ide/issues?q=)
- [X] I verified the problem still occurs when using the latest [nightly build](https://www.arduino.cc/en/software#nightly-builds)
- [X] My report contains all necessary details
